xray/backends/netCDF4_.py

```diff
@@ -54,7 +54,7 @@ def _nc4_values_and_dtype(var):
     return var, dtype
 
 
-def _nc4_group(ds, group):
+def _nc4_group(ds, group, mode):
```

Comment:
It'd be nice to have a way to list groups in a netCDF, to facilitate opening ... but that's probably future work.